src/server.jl

```diff
@@ -285,7 +288,9 @@ write_json(::Nothing, data) = data
 Return a vector of raw markdown and code chunks from `file` ready for evaluation
 by `evaluate_raw_cells!`.
 """
-raw_text_chunks(file::File) = raw_text_chunks(file.path)
+raw_text_chunks(file::File, ::Nothing) = raw_text_chunks(file.path)
+raw_text_chunks(file::File, markdown::String) =
+    raw_markdown_chunks_from_string(file.path, markdown)
```

Comment:
Does this handle `.jl` "script" files that can be passed to `quarto`?

Reply:
I assume those are preconverted by quarto as it's called `target.markdown`? But I can add a test

Reply:
hm tried to add this test:

````
# %% [markdown]
# ---
# engine: julia
# ---

# %%
println("the result is $(1 + 1)")
````

But quarto executed this using the jupyter engine. So that could be bug over there. I never tested script files so that's in the realm of possibilities

Reply:
I'll open an issue over there, but let's not have that hold up this PR

Reply:
https://github.com/quarto-dev/quarto-cli/issues/10034